src/Conf.cpp

```diff
@@ -358,6 +370,78 @@ void Conf::setDataCenter(const ConfParser::Node* node)
     }
 }
 
+void Conf::setCustomCommand(const ConfParser::Node* node)
+{
+    if (!node->sub) {
+        Throw(InvalidValue, "%s:%d CustomCommand require scope value", node->file, node->line);
+    }
+    for (auto p = node->sub; p; p = p->next) {
+        mCustomCommands.push_back(CustomCommandConf{});
+        auto& cc = mCustomCommands.back();
+        CustomCommandConf::init(cc, p->key.c_str(), Command::Sentinel);
+        auto s = p->sub;
+        for (;s ; s = s->next) {
+            if (setInt(cc.cmd.minArgs, "minArgs", s, 2)) {
+            } else if (setInt(cc.cmd.maxArgs, "maxArgs", s, 2, 9999)) {
+            } else if (setCommandMode(cc.cmd.mode, "mode", s)) {
+            } else {
+                Throw(UnknownKey, "%s:%d unknown key %s", s->file, s->line, s->key.c_str());
+            }
+        }
+        Command::addCustomCommand(&cc.cmd);
```

Comment:
Bug, The cc.cmd.name maybe point an invalid address when mCustomCommands push_back again

Reply:
I see and applied